.babelrc

```diff
@@ -1,7 +1,21 @@
 {
-    "presets": ["es2015", "flow"],
+    "presets": [
+      ["es2015", {
+        "modules": false
+      }],
+      "flow"
+    ],
     "plugins": [
         "transform-runtime",
         "transform-class-properties"
-    ]
+    ],
+    "env": {
+      "test": {
+        "presets": ["es2015", "flow"],
+        "plugins": [
```

Comment:
This is for Jest, as it [doesn't support ES6 modules](https://facebook.github.io/jest/docs/en/getting-started.html).

Babel has an issue merging env configurations in `.babelrc`(https://github.com/babel/babel/issues/5276), so plugins configuration has to be duplicated.